tools/Dockerfile

```
@@ -34,10 +34,10 @@ WORKDIR /stackable/bin
 ENV PATH=/stackable/bin:$PATH
 
 # Get latest stable version from curl -L -s https://dl.k8s.io/release/stable.txt
-RUN	curl -L https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl \
+RUN	curl https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl \
```

Replace tab with space

```suggestion
RUN curl https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl \
```

---

THERE'S A TAB IN THERE?

---

Do you have your tab size set to one space? I have mine set to 4 so it is obvious, not that I viewed this change locally)